pkgs/by-name/ya/yafc-ce/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDotnetModule,
+  pkgs,
```

```suggestion
  dotnetCorePackages,
  SDL2,
  SLD2_image,
  SDL2_ttd,
```
`pkgs` should not be used inside the Nixpkgs package set.